Rollup branch for updating SEO for HCP Terraform (#796)
hashicorp/terraform-docs-common: website/docs/cloud-docs/run/states.mdx

---
page_title: Run states and stages in HCP Terraform
description: >-

>   Learn the run stages that HCP Terraform goes through to perform Terraform operations. Understanding run stages and their states can help you follow a run's flow.

Reviewer: ```suggestion
  Learn the run stages of Terraform operations. Understanding run stages and their states can help you follow a run's progress.
```
